### ISSUE TITLE
Different Capacitor Settings per Platform

### ISSUE THREAD

**Issue by JensRavens** [ionic-team/capacitor#1117]
In a current project (using capacitor in production with version beta-8) I came across this problem:

- In android we're using the splashscreen with a static image. The launch sometimes is a big slow, therefore we increased the `SplashScreen->launchShowDuration` in the capacitor.config.json file.
- On iOS we use a custom launch screen. Therefore we would like to set the show duration of the plugin to 0 (or even better: remove the plugin completely as we don't need it). The capacitor launch screen would simply hide our handcrafted animation that we play while the webview is loading (and if we don't supply the image, just blocking the screen).

Currently it's not possible to set different arguments per platform, so everytime I `npx cap sync` the iOS specific changes on `capacitor.config.json` are  overwriten. I have to manually override it after every sync - which I tend to forget and just release a broken version one in a while.

Long story short: Is there any way to set platform specific options in the config and if not - can we turn this into a feature request?

**Comment by jcesarmobile** [comment]
Issues tagged with feature request are closed but tracked for reactions to gauge interest.
If you are interested in this feature, add the thumbs up emoji (+1) reaction on the issue so we can sort them based on it.

**Comment by jcesarmobile** [comment]
this is now possible as you can use a `capacitor.config.js` or `capacitor.config.ts` file instead of the old .json, so you can add any logic you want to set different values
